.gitignore

```diff
@@ -1,8 +1,9 @@
 .idea/
 .settings/
 hazelcast/downloaded
-hazelcast/hazelcast*
-hazelcast/*.zip
+hazelcast/hazelcast-*
+hazelcast/download.info
+hazelcast/*.zip*
```

Comment:
If you use `clean compile` the download process is triggered twice, so there is a `*.zip` and `*.zip.1` in the end.